src/utils.ts

```diff
@@ -27,10 +27,11 @@ export const rotr = (word: number, shift: number) => (word << (32 - shift)) | (w
 
 // big-endian hardware is rare. Just in case someone still decides to run hashes:
 // early-throw an error because we don't support BE yet.
-export const isLE = new Uint8Array(new Uint32Array([0x11223344]).buffer)[0] === 0x44;
+export const isLE =
+  /*#__PURE__*/ new Uint8Array(/*#__PURE__*/ new Uint32Array([0x11223344]).buffer)[0] === 0x44;
```

Comment:
probably doesn't make sense for this one? It's used immediately.

Reply:
I used babel plugin to add annotations. this one [babel-plugin-annotate-pure-calls](https://github.com/Andarist/babel-plugin-annotate-pure-calls), I didn't analyze which one makes sense and which is not, *all* top-level calls are marked as pure. I don't know is better for maintainability or not. What do you think?

Reply:
We won't be using babel in the package, because it would bring in a lot of dependencies, which we can't really audit. So while it makes sense to use babel as a starting point, it won't matter much in the future.